libsnark/zk_proof_systems/plonk/tests/test_plonk.cpp

```diff
@@ -1095,6 +1095,157 @@ template<typename ppT> void test_plonk_gates_matrix_transpose()
     ASSERT_EQ(gates_matrix_transpose, example.gates_matrix_transpose);
 }
 
+// We test the example circuit y^2 = x mod r where x is a public input
+// and y is the witness. We prove that we know y such that x is the
+// quadratic residue of y modulo the modulus r of the scalar field Fr
```

Comment:
"x is the quadratic residue of y" doesn't seem to be the correct terminology.
```suggestion
// and y is the witness.  Thus the circuit shows that x is a quadratic residue in the field Fr.
```

Reply:
Fixed